spec/rubocop/cop/rspec/named_subject_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::NamedSubject do
+  subject(:cop) { described_class.new }
+
+  it 'checks it for explicit subject usage' do
```

Comment:
Should this be "checks it and specify for explicit subject usage" ?

(also, perhaps enclose the keywords in backticks for emphasis and readability?).